src/game/default/g_cmd.c

```diff
@@ -662,6 +662,49 @@ static const char *vote_cmds[] = {
 		"unmute",
 		NULL };
 
+static GList *vote_cvars; // temp number so it doesn't have to be dynamically allocated
+
+/**
+ * @brief Create g_vote_allow_<vote> CVars
+ */
+void G_InitVote() {
+	size_t i = 0;
+
+	while (vote_cmds[i]) {
+		const char *name = va("g_vote_allow_%s", vote_cmds[i]);
+		const char *desc = va("Allows voting on %s", vote_cmds[i]);
+
+		vote_cvars = g_list_append(vote_cvars, gi.Cvar(name, "1", CVAR_SERVER_INFO, desc));
+
+		i++;
+	}
+}
+
+/**
+ * @brief Deinitialize the vote CVar GList
+ */
+void G_ShutdownVote() {
+  g_list_free(vote_cvars);
+}
+
+/**
+ * @brief Returns if the g_vote_allow_<vote> CVar is enabled
+ */
+static _Bool G_VoteAllowed(const char *vote) {
+	const char *name = va("g_vote_allow_%s", vote);
+
+	GList *list = vote_cvars;
+
+	while (list) {
+		if ((!g_strcmp0(name, ((cvar_t *)list->data)->name)) && ((cvar_t *) list->data)->integer)
```

Comment:
Cool. I would use a `const cvar_t *var = list->data;` here just for readability.